.github/workflows/validate-clean-linux-binaries.yml

```diff
@@ -0,0 +1,248 @@
+name: Validate Clean binary  images
+
+on:
+  push:
+    branches:
+      main
+    paths:
+      - .github/workflows/validate-clean-linux-binaries.yml
+  pull_request:
+    paths:
+      - .github/workflows/validate-clean-linux-binaries.yml
+
+env:
+  DOCKER_REGISTRY: "docker.io"
+  DOCKER_BUILDKIT: 1
+  DOCKER_ID: ${{ secrets.DOCKER_ID }}
+  DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
+  WITH_PUSH: ${{ github.event_name == 'push' }}
+
+jobs:
+  validate-linux-conda-cuda-binaries:
+    runs-on: ubuntu-20.04-m60
+    timeout-minutes: 360
+    strategy:
+      fail-fast: false
+      matrix:
+        cuda_version: ["11.7", "11.6", "11.3", "10.2"]
```

Comment:
here "cpu" can be one more value of cuda_version we will only have to use different runner for this.